modules/client-configuration/pages/supported-features-debian.adoc

```diff
@@ -46,13 +46,13 @@ The icons in this table indicate:
 | Works with {productname} Proxy | ``Salt`` | ``Salt``
 | Action chains | ``Salt`` | ``Salt``
 | Software crash reporting | icon:times[role="danger"] | icon:times[role="danger"]
-| Staging (pre-download of packages) |  icon:question[role="gray"] | icon:question[role="gray"]
+| Staging (pre-download of packages) |  ``Salt`` | ``Salt``
 | Duplicate package reporting | ``Salt`` | ``Salt``
 | CVE auditing |  icon:question[role="gray"] | icon:question[role="gray"]
 | SCAP auditing | icon:question[role="gray"] | icon:question[role="gray"]
 | Package verification | icon:times[role="danger"] | icon:times[role="danger"]
-| Package locking | icon:times[role="danger"] | icon:times[role="danger"]
-| System locking |  icon:times[role="danger"] | icon:times[role="danger"]
+| Package locking | ``Salt`` | ``Salt``
```

Comment:
AFAIK Package locking is not implemented for salt.

Reply:
Agreed, see above.